packages/editor/src/components/rich-text/index.js

```diff
@@ -392,11 +386,6 @@ export class RichText extends Component {
 	 * Handles the `selectionchange` event: sync the selection to local state.
 	 */
 	onSelectionChange() {
-		// Ensure it's the active element. This is a global event.
-		if ( ! this.isActive() ) {
```

Comment:
I think I'd noted it elsewhere, but there's a small difference between `onFocus` and `isActive` in that the prior was testing the `div` wrapper as the active element, the latter exclusively applied to the TinyMCE element (i.e. not considering toolbars and such). When last I tested this, I recall it being a reasonable difference in that (a) all other elements are non-visual or slot-rendered and (b) are not relevant to the selection anyways.

Does that sound accurate to you?

Reply:
Hm, I think both are checking the `contenteditable` element? `this.editableRef` is the `contenteditable` element, and `onFocus` is a listener on that element. Maybe you're thinking about `setFocusedElement`?

Reply:
> Hm, I think both are checking the `contenteditable` element? `this.editableRef` is the `contenteditable` element, and `onFocus` is a listener on that element. Maybe you're thinking about `setFocusedElement`?

Ah! I am!

I still think we should consolidate these, as having multiple concepts of focus here is particularly confusing for me, but it's not relevant to the change proposed here.

Reply:
Yeah, tbh I'm not sure why there's something like `setFocusedElement` on the wrapper...

Reply:
My hope was that at some point, we could get rid of the wrapper, at least in the sense of having it be a `<Fragment>` instead of an actual DOM `div` node.